third_party/move/move-vm/runtime/src/loader.rs

```diff
@@ -777,10 +778,14 @@ impl Loader {
 
     // Script verification steps.
     // See `verify_module()` for module verification steps.
-    fn verify_script(&self, script: &CompiledScript) -> VMResult<()> {
+    fn verify_script(&self, script: &CompiledScript, script_size: usize) -> VMResult<()> {
```

Comment:
Should we instead consider adding length to CompiledScript? It seems dangerous to decompose these.

Reply:
Yeah this is what I also found tricky. I also hope there's an easy way to obtain the size of a `CompiledModule/Script`, but the problem is that such size is not always well defined. `CompiledModule` by its current definition, is basically a low level AST of a Move module, and by because of this, (serialized) size is naturally doesn't belong to this struct and it could be undefined in many use cases (e.g. a test creating a module in memory).